.github/workflows/e2e-ci.yaml

```diff
@@ -376,7 +379,67 @@ jobs:
           architecture: '${{ secrets.ARCHITECTURE }}'
           iot_edge_deployment_id: '${{ secrets.IOT_EDGE_DEPLOYMENT_ID }}'
 
-# Runs E2E tests in dedicated agent, while having modules deployed into PI (arm32v7)
+  load_test_job:
+    timeout-minutes: 150
+    name: Run Load Tests
+    environment:
+      name: CI_AZURE_ENVIRONMENT
+      url: ${{ needs.env_var.outputs.CheckSuiteUrl }}
+    if: always() && (github.event_name == 'schedule' || github.event.inputs.RunLoadTests == 'true')
+    needs:
+      - deploy_arm_gw_iot_edge
+      - deploy_eflow_gw_iot_edge
+      - deploy_facade_function
+      - deploy_eflow_concentrator_iot_edge
+      - env_var
+      - certificates_job
+    runs-on: [ self-hosted, x64 ]
+    env:
+      INTEGRATIONTEST_IoTHubEventHubConnectionString: ${{ secrets.IOTHUB_EVENT_HUB_CONNECTION_STRING }}
+      INTEGRATIONTEST_IoTHubEventHubConsumerGroup: ${{ secrets.IOTHUB_CI_CONSUMER_GROUP }}
+      INTEGRATIONTEST_IoTHubConnectionString: ${{ secrets.IOTHUB_OWNER_CONNECTION_STRING }}
+      INTEGRATIONTEST_LeafDeviceGatewayID: itestarm1
+      INTEGRATIONTEST_DevicePrefix: '03'
+      INTEGRATIONTEST_RunningInCI: true
+      INTEGRATIONTEST_LoadTestLnsEndpoints: ${{ '[\"wss://itestarm1:5001\"]' }}
```

Comment:
Note: I will extract this to the `secrets` once I know if we need other configuration values that depend on the workflow runtime environment (before merging).

Reply:
Done with 3219e66a3c6c9e5ec34cdefc358d099360e6c6d1